api/coreservice.go

```diff
@@ -1194,7 +1194,7 @@ func (core *coreService) reverseActionsInBlock(blk *block.Block, reverseStart, c
 		selp := blk.Actions[ri]
 		actHash, err := selp.Hash()
 		if err != nil {
-			log.L().Debug("Skipping action due to hash error", zap.Error(err))
+			log.Logger("api").Debug("Skipping action due to hash error", zap.Error(err))
```

Comment:
there are 2 more log.L() in `reverseActionsInBlock`, should that be changed as well?